Config/editeurTerrain.py

```diff
@@ -0,0 +1,243 @@
+import tkinter as tk
+from typing import List
+
+from Config.tile import Tile
+
+"""
+Code to create the graph configuration file with a simple GUI. Parameters you may change:
+- number of columns (num_columns)
+- number of rows (num_rows)
+- the file name to save the file to (file_name)
+
+When the code is run, click on the midpoints of two adjacent segments. The configuration file is update automatically.
+The file can be then used in editeurConfig.py to specify other parameters.
+"""
+
+num_rows = 5
+num_columns = 5
+file_name = "graph_config.txt"
+
+
+# Grid initialization, (x,y) is inverted compared to (num_rows, num_columns)
+num_cells_x = num_columns
+num_cells_y = num_rows
+
+# Set the size of each cell in the grid
+cell_size = 80
+offset = cell_size
+
+current_point = None
+
+graph = [[Tile() for _ in range(num_cells_y)] for _ in range(num_cells_x)]
+
+def saveToFile(graph: List[List[Tile]],file_name: str) -> None:
+    """
+    Save given graph in file
+    :param graph: the graph (2d list of tiles)
+    :param file_name: the file name (str)
+    """
+    with open(file_name,"w") as f:
+        f.write("## DIMENSION ##\n")
+        f.write("row:{} column:{}\n".format(num_cells_y, num_cells_x))
```

Comment:
You can also use the more modern formatting approach:
```
f.write(f"row:{num_cells_y} column:{num_cells_x}\n")
```

Reply:
Corrected in commit c459d849aec6f0e6e979d0acd679c2dcc7114783